lib_vehicle_model/src/lib_vehicle_model/ModelLoader.cpp

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
It is a very good example on how to load libraries.

Reply:
Thanks